internal/publish/serverservice_components.go

```diff
@@ -817,7 +830,9 @@ type attributes struct {
 type versionedAttributes struct {
 	Firmware    *common.Firmware `json:"firmware,omitempty"`
 	Status      *common.Status   `json:"status,omitempty"`
+	UUID        *uuid.UUID       `json:"uuid,omitempty"`
```

Comment:
```suggestion
	UUID        *uuid.UUID       `json:"uuid,omitempty"` // UUID references firmware UUID identified in serverservice based on component/device attributes.
```
Comment on what the UUID here references would be helpful.